docs/button.html

```diff
@@ -0,0 +1,504 @@
+                        </li>
+                       </a>
+                       <a href="../docs/image.html" class="text-deocration-none selected-text ">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='images' color='#eeeded' ></box-icon>
+                            <span>Images</span>
+                        </li>
+                       </a>
+                       <a href="../docs/list.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='list-ul' color='#eeeded' ></box-icon>
+                            <span>Lists</span>
+                        </li>
+                       </a>
+                       <a href="../docs/modal.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='card' color='#eeeded' ></box-icon>
+                            <span>Modal</span>
+                        </li>
+                       </a>
+                       <a href="../docs/navigation.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='navigation' color='#eeeded' ></box-icon>
+                            <span>Navbars</span>
+                        </li>
+                       </a>
+                       <a href="../docs/rating.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='star-half' type='solid' color='#eeeded' ></box-icon>
+                            <span>Ratings</span>
+                        </li>
+                       </a>
+                       <a href="../docs/toast.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='copy' color='#eeeded' ></box-icon>
+                            <span>Toasts</span>
+                        </li>
+                       </a>
+                       <a href="../docs/typography.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='font-family' color='#eeeded' ></box-icon>
+                            <span>Typography</span>
+                        </li>
+                       </a>
+                    </ul>
+                        
+                 </div>
+            </div>
+             <div class="flex mt-1 ml-1 flex-align-item-center">
+                <svg width="40" height="32" viewBox="0 0 40 32" aria-hidden="true">
```

Comment:
try to have svg in a different file. i dont think its best practice to have it directly in the html file.

Reply:
+1